lib/src/visitor/async_evaluate.dart

```diff
@@ -2303,10 +2304,63 @@ class _EvaluateVisitor
         case "calc":
           assert(arguments.length == 1);
           return SassCalculation.calc(arguments[0]);
+        case "sqrt":
+          assert(arguments.length == 1);
+          return SassCalculation.sqrt(arguments[0]);
+        case "sin":
+          assert(arguments.length == 1);
+          return SassCalculation.sin(arguments[0]);
+        case "cos":
+          assert(arguments.length == 1);
+          return SassCalculation.cos(arguments[0]);
+        case "tan":
+          assert(arguments.length == 1);
+          return SassCalculation.tan(arguments[0]);
+        case "asin":
+          assert(arguments.length == 1);
+          return SassCalculation.asin(arguments[0]);
+        case "acos":
+          assert(arguments.length == 1);
+          return SassCalculation.acos(arguments[0]);
+        case "atan":
+          assert(arguments.length == 1);
+          return SassCalculation.atan(arguments[0]);
+        case "abs":
+          assert(arguments.length == 1);
+          return SassCalculation.abs(arguments[0]);
+        case "exp":
+          assert(arguments.length == 1);
+          return SassCalculation.exp(arguments[0]);
+        case "sign":
+          assert(arguments.length == 1);
+          return SassCalculation.sign(arguments[0]);
         case "min":
           return SassCalculation.min(arguments);
         case "max":
           return SassCalculation.max(arguments);
+        case "hypot":
+          return SassCalculation.hypot(arguments);
+        case "pow":
+          return SassCalculation.pow(
+              arguments[0], arguments.length > 1 ? arguments[1] : null);
```

Comment:
For all of these where you have `arguments.length > 1 ? arguments[1] : null`, I believe you can use dart's `elementAtOrNull` function:

e.g.
```dart
return SassCalculation.pow(arguments[0], arguments.elementAtOrNull(1));
```

Reply:
Yes, that makes sense. Just FYI: this isn't a built-in method on Dart iterables; it's an extension method from the collection package, so if you try to do this in the future and get errors, you just need to import `package:collection/collection.dart` (it's already imported for this file)